cookbooks/phase_transition_kinetics/phase-transition-kinetics.h

```diff
@@ -138,10 +138,20 @@ namespace aspect
         Utilities::AsciiDataProfile<dim> profile;
 
         /**
-         * Column indices from the thermodynamic data along an adiabatic profile.
-         * The pressure column is used for searching the table via interpolation.
-         * All other columns store thermodynamic data required for calculating
-         * the thermodynamic driving force in the PhaseTransitionKinetics material model.
+         * Path to the directory that stores the ascii data file.
+         */
+        std::string data_directory;
+
+        /**
+         * Name of the ascii data file.
+         */
+        std::string data_filename;
```

Comment:
this should not be necessary either, unless you want to use these variables yourself (outside of the `AsciiDataProfile` class).